README.md

```diff
@@ -187,7 +187,7 @@ example :
     ...
     Span5 = otter:span_ptag(Span4, "final result", "error"),
     Span6 = otter:span_ptag(Span5, "final result reason", "unknown user"),
-    otter:span_pend(Span6),
+    otter:span_end(Span6),
```

Comment:
also `otter:span_plog` and `otter:span_ptag` should be changed in the example too